nexus_constructor/nexus_filewriter_json/writer.py

```diff
@@ -204,3 +206,56 @@ def create_writer_commands(
         stop_cmd["stop_time"] = stop_time
 
     return write_cmd, stop_cmd
+
+
+def generate_forwarder_command(
```

Comment:
I don't think there is any reason to put these functions in this file.
Let's rename `nexus_filewriter_json` directory to something more general, maybe just `json`, and then rename `writer.py` to `filewriter_json_writer.py` or similar.